salt/metalk8s/addons/logging/fluent-bit/deployed/chart.sls

```diff
@@ -6,69 +6,32 @@
 
 {% raw %}
 
-apiVersion: policy/v1beta1
-kind: PodSecurityPolicy
-metadata:
-  labels:
-    app: fluent-bit
-    app.kubernetes.io/managed-by: salt
-    app.kubernetes.io/name: fluent-bit
-    app.kubernetes.io/part-of: metalk8s
-    chart: fluent-bit-2.2.0
-    heritage: metalk8s
-    release: fluent-bit
-  name: fluent-bit
-  namespace: metalk8s-logging
-spec:
-  allowPrivilegeEscalation: false
-  fsGroup:
-    rule: RunAsAny
-  hostIPC: false
-  hostNetwork: false
-  hostPID: false
-  privileged: false
-  readOnlyRootFilesystem: true
-  requiredDropCapabilities:
-  - ALL
-  runAsUser:
-    rule: RunAsAny
-  seLinux:
-    rule: RunAsAny
-  supplementalGroups:
-    rule: RunAsAny
-  volumes:
-  - secret
-  - configMap
-  - hostPath
-  - projected
-  - downwardAPI
----
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   labels:
-    app: fluent-bit
+    app.kubernetes.io/instance: fluent-bit
     app.kubernetes.io/managed-by: salt
     app.kubernetes.io/name: fluent-bit
     app.kubernetes.io/part-of: metalk8s
-    chart: fluent-bit-2.2.0
+    app.kubernetes.io/version: 1.8.12
+    helm.sh/chart: fluent-bit-0.19.19
     heritage: metalk8s
-    release: fluent-bit
   name: fluent-bit
   namespace: metalk8s-logging
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   labels:
-    app: fluent-bit
+    app.kubernetes.io/instance: fluent-bit
     app.kubernetes.io/managed-by: salt
     app.kubernetes.io/name: fluent-bit
     app.kubernetes.io/part-of: metalk8s
-    chart: fluent-bit-2.2.0
+    app.kubernetes.io/version: 1.8.12
+    helm.sh/chart: fluent-bit-0.19.19
     heritage: metalk8s
-    release: fluent-bit
-  name: fluent-bit-clusterrole
+  name: fluent-bit
```

Comment:
Names (for this ClusterRole, and the related binding) change, so we'll have some cleanup to add on upgrade

Reply:
Right, I did not check upgrade, I will take a look at it

Reply:
Upgrade and downgrade good now ! :smiley: